### PR TITLE
sorting issue #30

### PR DESCRIPTION
**Pull request by milanych**

### CODE REVIEW

**Review by Codeshark-NET — CHANGES_REQUESTED** [review]
Hey @milanych! Thank you kindly for adressing this issue 🙌 Overall the code looks good, and I'll go ahead and test it once the changes in this review have been adapted. Please use pnpm to follow the existing project structure. 👍 Thanks!

Update: You may also want to merge lastest commit on `main` 👍

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-pnpm lint
```
[the remark this answers is not in the document]
Comment (milanych) [reply .husky/pre-commit:4]: Hi, I am sorry to say that, but pull request seems to be the hardest part of this issue 🤦‍♂️
I've edited the pre-commit file to what it was before, and deleted package-lock, but...

If I try `git add .`, nothing happens.
If I try `git commit -m 'pr fixes'` it says something like 
```
C:\Users\username\AppData\Roaming\npm/node_modules/node/bin/node: line 1: This: command not found
husky - command not found in PATH=...
```
CONTRIBUTING.md literally says to do simple add and commit.

Sorry, this is the first "serious" pr, bear with me.
**So, what do I really need to do?**

[the remark this answers is not in the document]
Comment (Codeshark-NET) [reply .husky/pre-commit:4]: Don't be sorry - we all start somewhere. 😊

Your challenge may be a bit specific to your system, but here's a few things you can check:
- Make sure you have pnpm installed (you can install it globally via npm)
- Ensure that you added `pnpm lint` and `pnpm test` to the `pre-commit` file (you can test them both manually in your terminal to see if they fail or not)

Husky, in this case, is used as a pre-commit check, which runs those commands, before "allowing you" to commit. It's sort of like a safety check, to make sure that code commited is linted (and now with your addition, tested).

If the `pnpm test` addition is giving you problems, just don't include it for now.

**Comment by milanych** [comment]
@Codeshark-NET Morning, I finally fixed this Husky issue for my Win10. Hope everything is good now.

**Comment by Codeshark-NET** [comment]
> @Codeshark-NET Morning, I finally fixed this Husky issue for my Win10. Hope everything is good now.

Looks good @milanych , thanks! I'll test it out when I get a chance, and approve if everything is good. 👍

**Comment by Codeshark-NET** [comment]
Congrats on your first PR @milanych 🙌 I ran "prettier" on the code, to fix the linting errors, but otherwise great. 👍 Thanks a lot!

**Comment by Ly0n** [comment]
@Codeshark-NET @milanych I just realise that the "By project Age" filter is lost. :worried: . Any reason this was removed? I think this filter is relevant because some people want to contribute to a new project rather than fixing old ones.

**Comment by milanych** [comment]
@Codeshark-NET We have now 3 filters: by Age, by Downloads, and by Stars. What do you mean?

image

**Comment by Codeshark-NET** [comment]
The age there is the "most recent issues", I think @Ly0n is referring to the project age. Maybe we open a new issue to get that merged back in, if you're interested in doing that too @milanych :-)

**Comment by milanych** [comment]
So we need 4 buttons, right? "Recent", "Age", "Downloads", "Stars" ?

**Comment by Codeshark-NET** [comment]
> So we need 4 buttons, right? "Recent", "Age", "Downloads", "Stars" ?

Yes that sounds right 👍

**Comment by Ly0n** [comment]
I'm not sure if a new user will be able to understand these shortened versions. Maybe recent issues, project age, downloads and stars?

**Pull request merged**